examples/printerdemo/python/main.py

```diff
@@ -32,15 +34,15 @@ def quit(self):
 
     @slint.callback(global_name="PrinterQueue", name="start_job")
     def push_job(self, title):
-        self.printer_queue.append({
-            "status": "waiting",
-            "progress": 0,
-            "title": title,
-            "owner": "Me",
-            "pages": 1,
-            "size": "100kB",
-            "submission_date": str(datetime.now()),
-        })
+        self.printer_queue.append(PrinterQueueItem(
```

Comment:
Is this a breaking change or are both syntax supported?

Reply:
Its something that changed with https://github.com/slint-ui/slint/commit/e3aab79fdb999dd508459ed6058948647818faeb and this example wasn't adjusted (but also couldn't because this PR was missing).

Differently: the model contains structs, the code here https://github.com/slint-ui/slint/blob/49d0c96a9e1e734d320d55b2fa1e47064b801d67/examples/printerdemo/python/main.py#L53 expects it. But in the location you commented we put a dict into the model instead. So when fetching that row we'd get a dict but expected a struct.